packages/sdk-node/src/types.ts

```diff
@@ -34,6 +36,10 @@ export type ClerkMiddlewareOptions = {
   jwtKey?: string;
   strict?: boolean;
   signInUrl?: string;
+  /**
+   * @experimental
+   */
+  experimental_isApiRoute?: BooleanOrURLFnToBoolean;
```

Comment:
❓ Why not experimental_ignoreInterstitial?

Reply:
I thought that frameworks using the `@clerk/backend` would have the option to define their API routes (that's why I have set it as `experimental_isApiRoute `) and that we could use it internally to whatever we want eg:
- ignore interstitial
- avoid redirecting to sign-in (case of NextJS)
Mainly I have used the same logic we have for NextJS and applied it to sdk-node.
Should I rename it to `experimental_ignoreInterstitial` or not?
cc: @nikosdouvlis